src/components/merge_results/fields/CompareFields.tsx

```diff
@@ -0,0 +1,69 @@
+import styled from '@emotion/styled';
+import ArrowForwardIcon from '@mui/icons-material/ArrowForwardIos';
+import CheckIcon from '@mui/icons-material/Check';
+import RestoreIcon from '@mui/icons-material/Restore';
+import WarningAmberIcon from '@mui/icons-material/WarningAmber';
+import { Box, Button, Typography } from '@mui/material';
+import { ReactNode } from 'react';
+import { useTranslation } from 'react-i18next';
+
+interface CompareFieldsProps {
+  sourceContent: ReactNode;
+  targetContent: ReactNode;
+  isMatching: boolean;
+  isChanged: boolean;
+  onCopyValue: () => void;
+  onResetValue: () => void;
+}
+
+const StyledBox = styled(Box)({
+  padding: '0.1rem 0.5rem',
+  display: 'flex',
+  alignItems: 'center',
+  gap: '0.25rem',
+});
+
+export const CompareFields = ({
+  sourceContent,
+  targetContent,
+  isMatching,
+  isChanged,
+  onCopyValue,
+  onResetValue,
+}: CompareFieldsProps) => {
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <Box sx={{ p: '0.5rem', bgcolor: '#FEFBF3' }}>{sourceContent}</Box>
```

Comment:
Bruker samme farge her som vi har lagt inn for alternating colors på tabeller. Ikke optimalt å legge det inn her hardkodet, men har mistanke om at det er best nå likevel siden det går en prosess på siden med å endre på paletten(?)